tests/graphql/omit-many-list.graphql

```diff
@@ -0,0 +1,38 @@
+#> server-sdl
+schema @server @upstream {
+  query: Query
+}
+
+type A {
+  b: [String]
+  c: String
+  d: String
+}
+
+type Query {
+  u: U @http(baseURL: "http://jsonplaceholder.typicode.com", path: "/us/1")
+}
+
+type U
+  @addField(name: "b", path: ["a", "b"])
+  @addField(name: "c", path: ["a", "c"])
+  @addField(name: "d", path: ["a", "d"]) {
+  a: A @omit
```

Comment:
Can use a simpler example

Reply:
I've added this assertion on `omit-many.graphql` and deleted this one